src/traces/scattergl/index.js

```diff
@@ -590,32 +592,30 @@ function plot(gd, subplot, cdata) {
             null;
     });
 
-    if(selectMode) {
```

Comment:
Ah no. We need to keep that `selectMode` condition.

As creating one regl context takes about ~300ms, we don't want to create the selection regl context when in non-select interaction modes.

But perhaps, you only remove it before implementing `clickmode`. Please ignore if that's the case.

Reply:
Oh, I'll have to dig into that again. With `if(selectMode)` scattergl traces weren't updated on select-on-click when in zoom/pan dragMode. I guess I'll have to adapt the logic when `selectMode` flag is set to `true`.

Reply:
As far as I understand, the selection regl context is created by a call to `createScatter()`. When user switches to lasso or box mode, the plot is re-rendered for some reason and regl contexts are creating again.

I think you are absolutely right to take `clickMode` into account to decide if `scene.select2d` context needs to be created.

For now, I'll reintroduce the `selectMode` condition and add a TODO for later when I introduce `clickmode`.